app/controllers/projects_controller.rb

```diff
@@ -14,7 +13,7 @@ def create
     @project = Project.new project_params
     if @project.save
       flash[:success] = t "project.create_success"
-      redirect_to root_path
+      redirect_to projects_url
```

Comment:
```suggestion
      redirect_to projects_path
```
trong các controller chỉ cần dùng path là dc r